### PR TITLE
conf(FWS-91): Clean up files

### PR DESCRIPTION
**Pull request by emilegirardGit**
JIRA: link to jira ticket
https://champlainsaintlambert.atlassian.net/browse/FWS-91

Context:
This ticket was about cleaning unused files and imports.

Changes
-Cleaned frontend
-Cleaned backend

### CODE REVIEW

**Review by Abdur-Siddiqui — APPROVED** [review]
Looks good great work

File: src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentRequestModel.java [hunk 1]
```diff
@@ -11,16 +11,16 @@
...
-    private String serviceId;
-    private Status status;
-    private String location;
-    private String firstName;
-    private String lastName;
-    private String phoneNum;
-    private String date;
-    private String time;
+    String availabilityId;
+    String userId;
```
Comment (Eli293) [inline src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentRequestModel.java:15]: Is a there a reason these are no longer private?

**Review by Eli293 — APPROVED** [review]
Looks good.

File: src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentRequestModel.java (hunk 1 above)
Comment (emilegirardGit) [reply src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentRequestModel.java:15]: they are already public

**Pull request merged**